crates/graphql/src/schema/query_root.rs

```diff
@@ -475,6 +475,10 @@ impl QueryRoot {
         let conn = context.shared.db.get()?;
         let mut rows: Vec<models::StoreConfigJson> = store_config_jsons::table
             .filter(store_config_jsons::subdomain.eq(subdomain))
+            .filter(
+                store_config_jsons::store_address
+                    .ne_all(&context.shared.marketplaces_store_address_exclusions),
```

Comment:
```suggestion
                    .eq_any(&context.shared.marketplaces_store_address_exclusions),
```
"Filter out any records that are in the provided list".

More idiomatic to use positive assertions.

Reply:
Those are opposite operations. `eq_any` will return only those in the provided list. Are you asking for the following (i.e. add `not`)?

```
.filter(
                not(listing_receipts::seller
                    .eq_any(&context.shared.featured_listings_seller_exclusions)),
            )
```

Reply:
Also, in looking it up, [diesel says](https://docs.diesel.rs/diesel/expression_methods/trait.ExpressionMethods.html#method.ne_all) we should use `ne(all(...))` for postgres dbs

Reply:
Thank you sorry about the confusion on my end.